src/synology_dsm/api/surveillance_station/camera.py

```diff
@@ -66,35 +69,44 @@ def is_recording(self):
 class SynoCameraLiveView:
     """An representation of a Synology SurveillanceStation camera live view."""
 
-    def __init__(self, data):
+    def __init__(self, data, base_url):
         """Initialize a Surveillance Station camera live view."""
+        self._base_url = base_url
         self.update(data)
 
+    def __format(self, url):
```

Comment:
Can you please add typing?

Reply:
I'm good with this comment, but as I see other def in the file, there is no typing, so I'm ok with that.